README.md

```diff
@@ -15,23 +15,23 @@ In case we are missing any examples, feel free to [let us know](https://github.c
 3. Install [Gulp](http://gulpjs.com/) via `npm`:
    * *Note:* You may need to use `sudo` on the last command depending on your Node installation.
```

Comment:
nits: 

* I'd prefer to remove the bullet before Note (not a big fan of single bullet lists)
* remove "on the last command" as there is only one command left